### ISSUE TITLE
Weird particle distribution from Soul Brazier

### ISSUE THREAD

**Issue by Extegral** [lilypuree/Decorative-Blocks#34]
So, basically, soul particles spawned by Soul Brazier only move in positive X/positive Z directions, without even considering negative X/Y or entire Y axis. This should give a better presentation:

![image](https://user-images.githubusercontent.com/47505981/98748027-113c0480-23b9-11eb-9de5-af4bf43827d4.png)
I would make a pull request with fix for this, but it is a simple enough issue that is caused by this single line:
https://github.com/lilypuree/Decorative-Blocks/blob/d90fd0a2c980208e13ee15e77ad3d8a7b94040c4/src/main/java/com/lilypuree/decorative_blocks/blocks/BrazierBlock.java#L124

What that line should look like is something like this:
```
worldIn.addParticle(ParticleTypes.SOUL, pos.getX() + 0.5, pos.getY() + 0.8, pos.getZ() + 0.5, ((rand.nextDouble()-0.5)/10.0), (rand.nextDouble())/5.0), ((rand.nextDouble()-0.5)/10.0));
```

...assuming you don't mind getting rid of those weird double-float-double casts which really have no business being there, I would also advise to make the same change to lava particles for original brazier, it shouldn't matter because their velocity doesn't seem to be dependent on what you'd want it to be, but just to keep things organized.

**Comment by lilypuree** [comment]
It has been fixed for 1.16.1, and 1.16.2+ will recieve a fix once I finalize the 1.16.1 version so I'll close this issue now.

**Comment by Extegral** [comment]
👌
